samples/createNote.js

```diff
@@ -0,0 +1,35 @@
+async function main(
```

Comment:
Would you mind headers that show how to use these?  Example:

```js
// sample-metadata:
//   title: Export Assets
//   description: Export asserts to specified dump file path.
//   usage: node exportAssets.js <gs://my-bucket/my-assets.txt>
```

If you do this, it will show instructions on how to call the samples from the README.

Reply:
should the usage include params? for example:
```
// usage: node createNote.js "project-id" "note-id"
```

Reply:
Yes!  That's the whole idea - that way they know what to pass.